spirv_hlsl.cpp

```diff
@@ -2821,7 +2821,7 @@ void CompilerHLSL::emit_push_constant_block(const SPIRVariable &var)
 					const auto &member = type.member_types[i];
 
 					add_member_name(type, constant_index);
-					auto backup_name = get_member_name(type.self, i);
+					const auto &backup_name = get_member_name(type.self, i);
```

Comment:
Still broken.

Reply:
fix it https://github.com/KhronosGroup/SPIRV-Cross/pull/2245/commits/707f8c6e8395cb319c1abfd301d1bbed02637998